orchestra/static/dist/main.css

```diff
@@ -2320,25 +2320,31 @@ a.logo {
     .todo-list .new-todo__description__datetime {
```

Comment:
How does the line wrap when the description of a todo is long? Can you make a screenshot of that?

Reply:
Is it possible to make sure that the text don't wrap up under the box? Example: https://ps.w.org/cleverness-to-do-list/assets/screenshot-1.jpg?rev=1132235

Reply:
Fixed it. I also updated the screenshots.